### DISCUSSION TITLE
Local usage, e.g. to search Notes within Obsidian?

### DISCUSSION THREAD

**Discussion by tsteckenborn** [oramasearch/orama#671] [Ideas]
MiniSearch's integration with [Omnisearch](https://github.com/scambier/obsidian-omnisearch) for Obsidian demonstrates the potential for enhancing Obsidians search functionalities. Given this, would it be worth considering whether Orama could similarly be utilized to search through local notes using a plugin? Would there be benefits in doing this?

Has anyone explored the feasibility of integrating Orama for this purpose, and what steps would be necessary to implement it effectively?

**Comment by micheleriva** [comment]
Local storage is definitely an interesting use case. You can achieve this with the Data Persistence Plugin (https://docs.askorama.ai/open-source/plugins/plugin-data-persistence) but it would also be fun to develop a dedicated plugin for localstorage.
